recipes/pm4py/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "pm4py" %}
+{% set version = "2.2.19.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/ab/19/aa8e3014e9d606a9cf81dc435c4fd39aa89a3f57c6d64c55c730c36685b3/pm4py-2.2.19.1.tar.gz
```

Comment:
Use a more standard approach, either the tar.gz from pypi or the source code from github so that the bots can automatically update it